src/views/monitoring/alert-manager/store/actions.ts

```diff
@@ -16,7 +16,7 @@ export const getAlertData = async ({ commit }, alertId: string): Promise<void|Er
 
 export const updateAlertData = async ({ commit }, params: UpdateAlertParams): Promise<void|Error> => {
     try {
-        const alert: AlertDataModel = await SpaceConnector.client.monitory.alert.update({
+        const alert: AlertDataModel = await SpaceConnector.client.monitoring.alert.update({
             ...params.updateParams,
             alert_id: params.alertId,
         });
```

Comment:
한 가지 제안이 있습니다!
`UpdateAlertParams` 인터페이스를 `alert_id`가 required property가 될 수 있도록 지정하고, `updateAlertData` 액션의 `payload`로는 `UpdateAlertParams` 타입의 params 객체를 받으면 어떨까요~?
`updateParams`와 `alert_id`를 굳이 구분하지 않고요!ㅎㅎ

```typescript
interface UpdateAlertParams {
  alert_id: string;
 xxx?: xxx;
 xxx?: xxx;
}
```

```suggestion
        const alert: AlertDataModel = await SpaceConnector.client.monitoring.alert.update(params);
```

Reply:
아 저도 이거 생각했어요!! 은근 불편하더라구요.. 그런데 조금 명시적으로 하는 것도 나쁘지 않을 것 같아서 이렇게 했어요! 우선 다른 컴포넌트들 리팩토링해보면서 다시 생각해볼게요!!!